packages/core-utils/src/itinerary.js

```diff
@@ -26,6 +26,23 @@ export function isTransit(mode) {
   return transitModes.includes(mode) || mode === "TRANSIT";
 }
 
+export function isOptional(leg) {
+  return leg.boardRule === "mustPhone";
+}
+export function isContinuousDropoff(leg) {
```

Comment:
Add JSDoc explaining why only "Must coordinate with driver" is used and not other continuous dropoff values.

Reply:
Added in f7aec8908db009cc5d1b7ddbfe2e062539692f51